content/services/csv/aarch64_open_source_projects.csv

```diff
@@ -17,6 +17,7 @@ compatible with RHEL® and pre-CentOS-Stream CentOS™ (e.g. CentOS 8.3). Founde
 "OpenFaaS","Making Serverless Functions Simple. Troubleshooting and development of ARM64 support for builds (to create artifacts) and function templates."
 "Open Mandriva", "Powerful and state of art, Linux distribution focused on desktop use"
 "OSMC", "A media centre for people. Play everything without collecting any data."
+"RebornOS", "Bring RebornOS to AARCH64"
```

Comment:
It might be good to add something else to this description. Can you see if you can come up with something?

Reply:
Updated it to paraphrase the About section on their website.